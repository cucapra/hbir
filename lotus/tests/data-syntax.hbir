target{
}

config{
}

data{
  dim = 1;
  
  in A : int[3][3] {
    location : config.t,
    layout : block
  } 

  out B : int[3][3] {
    location : config.t,
    layout : block
  }
}

code{
}
