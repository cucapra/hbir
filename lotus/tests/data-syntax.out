#include "bsg_manycore.h"
#include "bsg_set_tile_x_y.h"
#define dim 1

int A[3][3] __attribute__ ((section (".dram")));


